pkg/controller/utils/status.go

```diff
@@ -16,7 +16,7 @@ func SetAccountStatus(awsAccount *awsv1alpha1.Account, message string, ctype aws
 		UpdateConditionNever,
 		awsAccount.Spec.BYOC,
 	)
-	awsAccount.Status.State = state
+	awsAccount.Spec.State = state
```

Comment:
In this func we're passing in a state of type string, in the bellow func we're passing in a state of type `awsv1alpha1.ClaimStatus` -> can we update this func to be the same (or whatever the equivalent const type it should be)